### PR TITLE
Aswathy/signup affiliate success version-2 and validation

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation updateathy/ 
-   [ ] Release

### CODE REVIEW

File: src/pages/signup-affiliates/index.tsx [hunk 1]
```diff
@@ -117,88 +128,229 @@ const FacebookWrapper = styled.div``
...
+                    <EmailWrapper>
+                        <ImageWrapper>
+                            <img src={Email} alt="email" width="116" height="100" />
+                        </ImageWrapper>
+                        <Header type="subtitle-1" align="center">
+                            {localize('Check your email')}
+                        </Header>
+                        <Header type="paragraph-1" align="center" weight="400" pt="8px">
+                            {localize(
+                                "We've sent a message to test@deriv.com with a link to activate your account.",
```
Comment (prince-deriv) [inline src/pages/signup-affiliates/index.tsx:346]: We need to make the email dynamic

Comment (aswathy-deriv) [reply src/pages/signup-affiliates/index.tsx:346]: okay

File: src/pages/signup-affiliates/index.tsx [hunk 2]
```diff
@@ -117,88 +125,168 @@ const FacebookWrapper = styled.div``
...
+    flex-direction: column;
+`
+const ImageWrapper = styled.div`
+    margin: auto;
+`
 
-const AffiliateSignup = ({ autofocus, email, email_error_msg }: AffiliateSignupProps) => {
+const AffiliateSignup = () => {
+    const [email, setEmail] = useState('')
+    const [is_submitting, setSubmitting] = useState(false)
```
Comment (prince-deriv) [inline src/pages/signup-affiliates/index.tsx:138]: can we rename to `is_submitted`?

Comment (aswathy-deriv) [reply src/pages/signup-affiliates/index.tsx:138]: okay

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**